PendingReleaseNotes

```diff
@@ -43,6 +43,10 @@
   there are alternative monitoring solutions, like `prometheus`, which is the most
   widely adopted among the Ceph user community.
 
+* CephFS: EOPNOTSUPP (Operation not supported ) is now returned by the CephFS
+  fuse client for `fallocate` for the default case (i.e. mode == 0) since
```

Comment:
@mchangir - Sorry for the late remark but I just found out about this change. This change is causing a failure in the go-ceph ci (see https://github.com/ceph/go-ceph/issues/1044 ). The change itself is not a problem, we will deal with it. My request is to tweak the release note because 'fuse client' is a bit misleading as this change doesn't only impact fuse. Maybe tweak it to say 'fuse client or other libcephfs clients'?  Thanks.